addons/sale_project/wizard/project_template_create_wizard.py

```diff
@@ -12,3 +12,25 @@ def _get_template_whitelist_fields(self):
         if self.allow_billable:
             res.append("partner_id")
         return res
+
+    def _create_project_vals(self):
+        return {'name': self.name}
+
+    def action_create_project_from_so(self):
+        self.ensure_one()
+        project_vals = self._create_project_vals()
+        if self.template_id:
+            project = self.template_id.action_create_from_template(values=project_vals)
+        else:
+            project = self.env['project.project'].create(project_vals)
+        action = self.env['ir.actions.act_window'].with_context(active_id=project.id)._for_xml_id('project.act_project_project_2_project_task_all')
+        action['domain'] = [('project_id', '=', project.id)]
+        action['context'] = {'default_project_id': project.id}
+        return action
+
+    @api.model
+    def action_open_sale_project_view(self):
+        action = self.action_open_template_view()
+        view = self.env.ref('sale_project.sale_project_view_form_simplified_template', raise_if_not_found=False)
+        action['views'] = [(view.id, 'form')]
+        return action
```

Comment:
I don’t think there is a need to add this method. You can use action_open_template_view and the view  in the action_create_project method